website/pages/style.css

```diff
@@ -59,12 +54,12 @@
   padding-bottom: 96px;
 
   & > .g-section-header + *,
-  & > .g-container > .g-section-header + * {
+  & > .g-grid-container > .g-section-header + * {
     margin-top: 72px;
   }
 
   &:not(.no-section-spacing) > * + *,
-  & > .g-container > * + * {
+  &:not(.g-featured-slider-section) > .g-grid-container > * + * {
```

Comment:
Replacing `g-container` with `g-grid-container` had the unexpected side-effect of `featured-slider` receiving buggy spacing due to these styles. I think we should consider these global-level `* + *` styles to be debt - it seems with CSS modules in place, we could come up with a more composable way to use these styles on `section` elements, rather than having all `section` elements inherit this behaviour. Have created [a task to capture this](https://app.asana.com/0/1100423001970639/1200768863236365/f)